README.md

```diff
@@ -105,6 +105,14 @@ model = LLM("TinyLlama-1.1B-Chat-v1.0-INT8")
 output = model.generate("My name is")
 ```
 
+## New Features
+
+We've recently added several exciting features to LLM Compressor:
+
+* **Axolotl Sparse Finetuning Integration:** Easily finetune sparse LLMs through our integration with Axolotl. [Learn more here](https://docs.axolotl.ai/docs/custom_integrations.html#llmcompressor).
+* **AutoAWQ Integration:** Perform low-bit weight-only quantization efficiently using AutoAWQ, now part of LLM Compressor. [See the details](https://github.com/vllm-project/llm-compressor/pull/1177).
```

Comment:
I think we should call out that this should be considered experimental or highlight that support for MoE models or larger models (by moving to layer sequential pipelining) will be added soon

Reply:
Updated!